### ISSUE TITLE
Redis deprecation notice

### ISSUE THREAD

**Issue by bgwilson87** [deanpcmad/sidekiq-limit_fetch#123]
redis.multi do
  redis.get("key")
end

should be replaced by

redis.multi do |pipeline|
  pipeline.get("key")
end

(called from /Users/me/.rvm/gems/ruby-2.7.4/bundler/gems/sidekiq-limit_fetch-bf9cee3b81e4/lib/sidekiq/limit_fetch/global/monitor.rb:49:in `block in update_heartbeat'}
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

**Comment by yrashk** [comment]
Same here.

**Comment by iurev** [comment]
I submitted a new pull request fixing this issue:
https://github.com/brainopia/sidekiq-limit_fetch/pull/124

Another temporary way to fix this warning would be to downgrade to redis v4.5.1 where `multi` method works as usual. [link](https://github.com/redis/redis-rb/tree/v4.5.1#pipelining)

**Comment by rgaufman** [comment]
Please merge, we are seeing this as well

**Comment by frantisekrokusekpa** [comment]
Same here guys!

**Comment by mkolenda** [comment]
Same here too.  Thanks for fixing.

**Comment by rgaufman** [comment]
Any update please?

**Comment by iurev** [comment]
Unfortunately, this project is not currently supported and is looking for a new maintainer [https://github.com/brainopia/sidekiq-limit_fetch/issues/106](link)

As a temporary option, you may use version from my pull-request in your Gemfile.

_But beware, fetching gems directly from a stranger's github is a rather insecure practice._

```
gem 'sidekiq-limit_fetch',
  git: 'https://github.com/iurev/sidekiq-limit_fetch',
  branch: 'feature/fix_deprecated_redis_multi'
```

**Comment by deanpcmad** [comment]
Hey, I've now taken over this project and gem. I'll be going through the pull requests and issues in the next week or two :)

**Comment by deanpcmad** [comment]
This has been fixed with #124 - I'll try and get a release done this weekend
